fmn/consumer/backend.py

```diff
@@ -26,6 +23,8 @@
 import fmn.consumer.fmn_fasshim
 from fedmsg_meta_fedora_infrastructure import fasshim
 
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
On the one hand, yes, logging should be configured as soon as possible. On the other hand, imports should definitely _not_ have side-effects that require debug-level logging.

I didn't want to make any changes that were not style-related (or interpreter errors). My longer-term plan for this is to drag all the application setup into a Twisted ``tac`` file where logging is configured and the application is defined. After that, every module should simply import logging and get its configured logger with ``logging.getLogger(__name__)`` rather than passing a logger instance around.